<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextElement</class>
 <widget class="QWidget" name="TextElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Element Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Margins</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="MarginTop">
        <property name="toolTip">
         <string>Top margin in mm, internal to the border.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="MarginLeft">
        <property name="toolTip">
         <string>Left margin in mm, internal to the border.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="MarginRight">
        <property name="toolTip">
         <string>Right margin in mm, internal to the border.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="MarginBottom">
        <property name="toolTip">
         <string>Bottom margin in mm, internal to the border.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Border</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="ShowBorder">
        <property name="toolTip">
         <string>Show border around the text box</string>
        </property>
        <property name="text">
         <string>Show border</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string comment="Label for color value">Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KColorCombo" name="BorderColor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>The border color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Line thickness</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="BorderThickness">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Thickness of the border in 1/10 mm</string>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5" colspan="6">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string comment="Groupbox title for background properties">Background</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="FillBackground">
        <property name="toolTip">
         <string>Fill the background with a color</string>
        </property>
        <property name="text">
         <string>Fill background</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string comment="Label for color value">Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KColorCombo" name="BackgroundColor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Background color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Transparency</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="BackgroundTransparency">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Transparency of the text box</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFontComboBox" name="FontFamily">
     <property name="toolTip">
      <string>Text font family</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="PointSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Text point size</string>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>138</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="BoldButton">
     <property name="toolTip">
      <string>Bold text</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="ItalicButton">
     <property name="toolTip">
      <string>Italics text</string>
     </property>
     <property name="text">
       <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="UnderlineButton">
     <property name="toolTip">
      <string>Underline text</string>
     </property>
     <property name="text">
       <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QToolButton" name="AlignLeft">
     <property name="toolTip">
      <string>Align left</string>
     </property>
     <property name="text">
       <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QToolButton" name="AlignHCenter">
     <property name="toolTip">
      <string>Align centrally</string>
     </property>
     <property name="text">
       <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QToolButton" name="AlignRight">
     <property name="toolTip">
      <string>Align right</string>
     </property>
     <property name="text">
       <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QToolButton" name="AlignJustify">
     <property name="toolTip">
      <string>Align justified</string>
     </property>
     <property name="text">
       <string notr="true">...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QToolButton" name="TextColor">
     <property name="toolTip">
      <string>Text color</string>
     </property>
     <property name="text">
       <string notr="true">...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="11">
    <widget class="KTextEdit" name="Text">
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ShowBorder</sender>
   <signal>toggled(bool)</signal>
   <receiver>BorderColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ShowBorder</sender>
   <signal>toggled(bool)</signal>
   <receiver>BorderThickness</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FillBackground</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FillBackground</sender>
   <signal>toggled(bool)</signal>
   <receiver>BackgroundTransparency</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
